pixeltable/client.py

```diff
@@ -147,6 +150,174 @@ def create_table(
         _logger.info(f'Created table {path_str}')
         return tbl
 
+    def create_table_from_parquet(self, path_str: str, parquet_path: str,
```

Comment:
Not clear we should pile all of this into the client. It's going to get very big.

Reply:
yeah, moved.